os/src/Path.scala

```diff
@@ -435,7 +440,21 @@ object Path {
       }
     }
   }
-
+  def rootRelative[T: PathConvertible](f0: T): Boolean = {
+    if (currentWorkingDrive.isEmpty) {
+      false // non-Windows os
+    } else {
+      f0.toString.take(1) match {
+        case "\\" | "/" => true
+        case _ => false
+      }
+    }
+  }
+  def currentWorkingDrive: String = Paths.get("").toAbsolutePath.getRoot.toString match {
+    case "/" => ""
+    case s => s"$s/"
+  }
+  def currentDrive: Path = Path(currentWorkingDrive)
```

Comment:
Isn't this semantically the same as `root` (or `currentRoot`)?

It may sound as nitpicking, but a "drive" is a Windows-specific term (each drive resembles it own filesystem root), but we may even don't hold a drive letter at all.

Reply:
You are correct, they are the same, although using `os.root` in `os/src/Path.scala` causes circular references at compile time.